src/core/sagas/utils.js

```diff
@@ -1,9 +1,7 @@
-/* @flow */
```

Comment:
One thing I find really nice about annotating the `state` object is that you know exactly what you're getting. You can follow `ApiStateType` and find out what all the `state.api.*` properties are that you get access to. This would be hugely informative to an ES6 newcomer who is trying to understand how our app works. In fact, they could be pretty lost without it! Obviously it's not helpful in the case of `state.auth` because we don't have types for that one yet.